modules/terraform/show.go

```diff
@@ -0,0 +1,26 @@
+package terraform
+
+import (
+	"github.com/gruntwork-io/terratest/modules/testing"
+	"github.com/stretchr/testify/require"
+)
+
+// Show runs terraform show with the given options and returns stdout/stderr.
+// This will fail the test if there is an error in the command.
+func Show(t testing.TestingT, options *Options) string {
+	out, err := ShowE(t, options)
+	require.NoError(t, err)
+	return out
+}
+
+// ShowE calls terraform show for the given plan output file and returns it.
```

Comment:
Hm, shouldn't we respect booth versions of `show`? The one that just outputs the state file and the one that reads a `plan` file and outputs data about that?

Reply:
Fixed to allow running show without plan files: 7a3339e6ae1b0a031913a09312c420f35bce923e